assesment.js

```diff
@@ -0,0 +1,82 @@
+document.addEventListener("DOMContentLoaded",() =>{
+    console.log("page loaded");
+    fetchMovies()
+    SelectMovie()
+
+})
+
+function fetchMovies(){
+    const url = "https://ghibliapi.herokuapp.com/films"
+    // axios.get(url).then((response)=>{
+    //     response.data
+    //     })
+    
+    fetch(url).then(response =>{
+        return response.json() 
+        //console.log(response)
+    })
+    .then(data => {
+        let pick = data
+        console.log(pick[0])
+        let selection = document.querySelector("select");
+        
+        for(movie in pick){
+            let title = document.createElement("option");
+            title.value = pick[movie].title;
+            title.innerText = pick[movie].title;
+            selection.appendChild(title);
+
+        }
+    })
+}
+
+function SelectMovie(){
+    
+    const url = "https://ghibliapi.herokuapp.com/films"   
+    fetch(url).then(response =>{
```

Comment:
You could have stored the film `id` in the `optionElement.value` so that when the user picks an option `selectElement.value` will have the id. [Check this](https://www.w3schools.com/jsref/prop_select_value.asp) click `Try it yourself`. This would have prevented you from having to make another network request to `/films`. Instead you would make a request to `/films/<film-id>` which was our intent.
* index.js:1 Our intent here was for you to make a network request to `/films/<film-id>` after grabbing the id of the selected movie. How would you have done it if you couldn't use your global variable `filmsObject`. Think of this: you could have stored the film `id` in the `optionElement.value` so that when the user picks an option `selectElement.value` will have the id. [Check this](https://www.w3schools.com/jsref/prop_select_value.asp) click `Try it yourself`. Then make a second network request to `/films/<film-id>` to get the details of the movie.